src/libtools/property.cc

```diff
@@ -73,6 +75,10 @@ const Property &Property::get(const string &key) const {
   return *p;
 }
 
+Property &Property::get(const string &key) {
+  return const_cast<Property &>(static_cast<const Property &>(*this).get(key));
```

Comment:
Could you please briefly explain to me why is this double-casting needed? :)

Reply:
Basically it removes all the `const` , I think it was inspired by stack overflow.